packages/extension-ui/src/Popup/CreateAccount/index.tsx

```diff
@@ -51,11 +51,13 @@ export default function CreateAccount (): React.ReactElement {
           <ActionText text={step === 1 ? 'Cancel' : 'Back'} onClick={_onCancel} />
         </CreationSteps>
       </Header>
-      <Loading>{account && (step === 1 ? (
-        <Mnemonic seed={account.seed} onNextStep={_onNextStep} />
-      ) : (
-        <AccountName address={account.address} onCreate={_onCreate} />
-      ))}</Loading>
+      <Loading>
+        {account && (step === 1 ? (
+          <Mnemonic seed={account.seed} onNextStep={_onNextStep} />
+        ) : (
+          <AccountCredentials seed={account.seed} onCreate={_onCreate} />
+        ))}
```

Comment:
```suggestion
        {account && (step === 1)
          ? <Mnemonic seed={account.seed} onNextStep={_onNextStep} />
          : <AccountCredentials seed={account.seed} onCreate={_onCreate} />
        }
```

Just small refactoring, easier to read (imho) and fits in what we have elsewhere in this org.